src/cache/cache.ts

```diff
@@ -10,12 +10,14 @@ export interface CacheRequestPayload extends WithGroup {
   returnEvent: string;
 }
 
+/** The recommended prefix to use for the name of the event fired when data is returned from the cache. */
+export const CACHE_RESPONSE_PREFIX = 'sfx::cache_response-';
```

Comment:
should this be 'gbe' ?

Reply:
Not yet. The rest of the events in this file are still `sfx`. If this goes in first, it will be the responsibility of the Elements PR to change this. If the Elements PR goes in first, then it will be this PR's responsibility to follow suit.